src/ansys/additive/core/microstructure.py

```diff
@@ -42,43 +42,60 @@ class MicrostructureInput:
+    MIN_POSITION_COORDINATE = 0
+    """Minimum x, y, z, position coordinate (m)."""
+    MAX_POSITION_COORDINATE = 10
+    """Maximum x, y, z, position coordinate (m)."""
     DEFAULT_SAMPLE_SIZE = 1.5e-3
-    __MIN_SAMPLE_SIZE = 0.001
-    __MAX_SAMPLE_SIZE = 0.01
-    #: Default sensor dimension (m).
+    """Default sample size (m) in each dimension."""
+    MIN_SAMPLE_SIZE = 0.001
+    """Minimum sample size (m) in each dimension."""
+    MAX_SAMPLE_SIZE = 0.01
+    """Maximum sample size (m) in each dimension."""
     DEFAULT_SENSOR_DIMENSION = 5e-4
-    __MIN_SENSOR_DIMENSION = 1e-4
-    __MAX_SENSOR_DIMENSION = 1e-3
-    __MIN_XY_SIZE_CUSHION = 5e-4
-    __MIN_Z_SIZE_CUSHION = 1e-3
-    #: Default flag value indicating whether to use user provided thermal parameters.
+    """Default sensor dimension (m)."""
+    MIN_SENSOR_DIMENSION = 1e-4
+    """Minimum sensor dimension (m)."""
+    MAX_SENSOR_DIMENSION = 1e-3
+    """Maximum sensor dimension (m)."""
+    MIN_XY_SIZE_CUSHION = 5e-4
+    """Minimum cushion between sensor dimension and sample size in X and Y dimensions (m)."""
+    MIN_Z_SIZE_CUSHION = 1e-3
+    """Minimum cushion between sensor dimension and sample size in Z dimension (m)."""
     DEFAULT_USE_PROVIDED_THERMAL_PARAMETERS = False
-    #: Default cooling rate (K/s).
+    """Default flag value indicating whether to use user provided thermal parameters."""
     DEFAULT_COOLING_RATE = 1e6
-    __MIN_COOLING_RATE = 1e5
-    __MAX_COOLING_RATE = 1e7
-    #: Default thermal gradient (K/m).
+    """Default cooling rate (K/s)."""
+    MIN_COOLING_RATE = 1e5
+    """Minimum cooling rate (K/s)."""
+    MAX_COOLING_RATE = 1e7
+    """Maximum cooling rate (K/s)."""
     DEFAULT_THERMAL_GRADIENT = 1e7
-    __MIN_THERMAL_GRADIENT = 1e5
-    __MAX_THERMAL_GRADIENT = 1e8
-    #: Default melt pool width (m).
+    """Default thermal gradient (K/m)."""
+    MIN_THERMAL_GRADIENT = 1e5
+    """Minimum thermal gradient (K/m)."""
+    MAX_THERMAL_GRADIENT = 1e8
+    """Maximum thermal gradient (K/m)."""
     DEFAULT_MELT_POOL_WIDTH = 1.5e-4
-    __MIN_MELT_POOL_WIDTH = 7.5e-5
-    __MAX_MELT_POOL_WIDTH = 8e-4
-    #: Default melt pool depth (m).
+    """Default melt pool width (m)."""
+    MIN_MELT_POOL_WIDTH = 7.5e-5
+    """Minimum melt pool width (m)."""
+    MAX_MELT_POOL_WIDTH = 8e-4
+    """Maximum melt pool width (m)."""
     DEFAULT_MELT_POOL_DEPTH = 1e-4
-    __MIN_MELT_POOL_DEPTH = 1.5e-5
-    __MAX_MELT_POOL_DEPTH = 8e-4
-    #: The default random seed is outside the range of valid seeds.
-    #: It indicates that the user did not provide a seed.
+    """Default melt pool depth (m)."""
+    MIN_MELT_POOL_DEPTH = 1.5e-5
+    """Minimum melt pool depth (m)."""
+    MAX_MELT_POOL_DEPTH = 8e-4
+    """Maximum melt pool depth (m)."""
     DEFAULT_RANDOM_SEED = 0
-    __MIN_RANDOM_SEED = 1
-    __MAX_RANDOM_SEED = 2**31 - 1
+    """The default random seed, indicates that a random seed was not provided."""
```

Comment:
```suggestion
    """The default random seed, which indicates that a random seed was not provided."""
```
I'm not sure I fully understand this sentence. Could it be rewritten as: The default random seed, which is used if a random seed is not provided.